queries/cpp/highlights.scm

```diff
@@ -39,6 +44,8 @@
 (alias_declaration
   name: (type_identifier) @type.definition)
 
+(auto) @type.builtin
```

Comment:
auto is a builtin language tool to command compiler to decide the type so it's I'm open to create it's own hl group like @auto because it's a bit complicated to make it's own color in color scheme.